samples/44.prompt-users-for-input/README.md

```diff
@@ -0,0 +1,90 @@
+# Multi-turn prompt
```

Comment:
Update README with correct sample name, jar, etc... Compare with dotnet version for other verbiage changes.  Note the steps to deploy are different right now for Java (until docs are done).